src/System.Net.NameResolution/tests/PalTests/NameResolutionPalTests.cs

```diff
@@ -32,6 +32,17 @@ public void GetHostByName_LocalHost()
             Assert.NotNull(hostEntry.Aliases);
         }
 
+        public static object[][] InvalidHostNames = new object[][] {
+            new object[] { "fkdsfsjfsrH:" },
```

Comment:
>Maybe use Guid.NewGuid().ToString("N") instead?

Shortened to just ":"

FWIW I found that using a guid or "..." will result in `err` being non-zero on Ubuntu (so no seg fault) but ":" will cause `err` to be zero and `getHostErrno` to be non-zero and result in the seg fault.